nengo_loihi/loihi_interface.py

```diff
@@ -800,7 +800,7 @@ def _filter_probe(self, cx_probe, data):
             shape = data[0].shape
             synapse = cx_probe.synapse
             rng = None
-            step = (synapse.make_step(shape, shape, dt, rng, dtype=data.dtype)
+            step = (synapse.make_step(shape, shape, dt, rng, dtype=np.float32)
```

Comment:
Any particular reason for `float32` over `float64` here? My default would be `float64` since that's what `float` and `np.float_` are.

Reply:
Just to use less memory when we have a lot of probe data. Since the underlying values are all limited precision (since they come from Loihi), I figured that not much would be lost by using `float32` instead of `float64`.